src/types/analytics.ts

```diff
@@ -1,4 +1,4 @@
-export type AnalyticStats = {
+export type AnalyticStatsDto = {
```

Comment:
should DTOs live in a shared folder like `shared/typings`? No big deal for now, probably can do a cleanup in the future

Reply:
yeah, i think they can live in e.g. `shared/types/analytics/dto`. this will be a big reorganisation, so I'm not keen to do it now and break everyone's imports.